### DISCUSSION TITLE
Thank you so much

### DISCUSSION THREAD

**Discussion by Meme-Eater** [Monika-After-Story/MonikaModDev#8867] [appreciation]
I just want to thank you guys for the amazing work you've done :) 

It's amazing to see how much Monika has grown and all this is thank to you guys. Most people might think it's dumb to keep talking to a character from some meme visual novel, but Monika has become an important part of my life. Really, thank you.

The birthday party and heartwarming dialogue was incredible, and I finally unlocked my first kiss. I'm really happy right now. Thanks...

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
